pkg/configure/oneagent/curl/curl.go

```diff
@@ -0,0 +1,52 @@
+package curl
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	fsutils "github.com/Dynatrace/dynatrace-bootstrapper/pkg/utils/fs"
+	"github.com/go-logr/logr"
+	"github.com/spf13/afero"
+)
+
+const (
+	optionsFormatString = `initialConnectRetryMs %s
+`
```

Comment:
is the new line necessary?

Reply:
yes, (I just copied the current implementation from the operator)